db_stress_tool/db_stress_test_base.h

```diff
@@ -41,8 +42,24 @@ class StressTest {
   virtual void VerifyDb(ThreadState* thread) const = 0;
   virtual void ContinuouslyVerifyDb(ThreadState* /*thread*/) const = 0;
   void PrintStatistics();
+  bool MightHaveDataLoss() {
+    return FLAGS_sync_fault_injection || FLAGS_write_fault_one_in > 0 ||
+           FLAGS_metadata_write_fault_one_in > 0 || FLAGS_disable_wal ||
+           FLAGS_manual_wal_flush_one_in > 0;
```

Comment:
More conditions are added since now we inject write (so use `TestFSWritableFile`) and metadata write error to WAL file and WAL file folder.